osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -133,6 +132,8 @@ public void ShowUser(IUser user)
 
         private void userLoadComplete(APIUser user)
         {
+            Debug.Assert(sections != null && sectionsContainer != null && tabs != null);
```

Comment:
A bit of a weird one because this overlay does its drawable initialisation in `ShowUser()` rather than `load()` or anything of the sort, so there were existing null checks and the like. So I just rolled with that and added an assertion here.